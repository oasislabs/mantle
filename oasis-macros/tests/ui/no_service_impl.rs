#![feature(proc_macro_hygiene)]
#[oasis_std::service]
mod service {
    #[derive(Service)]
    pub struct Counter(u32);
}

fn main() {}
