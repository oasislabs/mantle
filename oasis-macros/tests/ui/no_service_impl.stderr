error: Missing implementation for `Counter::new`.
 --> $DIR/no_service_impl.rs:5:16
  |
5 |     pub struct Counter(u32);
  |                ^^^^^^^

error: aborting due to previous error

